pkg/nsx/services/vpc/vpc.go

```diff
@@ -69,31 +59,40 @@ type AVIAllowRule struct {
 	PubIpblockStore     *PubIPblockStore
 }
 
+func (s *VPCService) GetDefaultNetworkConfig() (bool, *VPCNetworkConfigInfo) {
+	for _, v := range s.VPCNetworkConfigMap {
```

Comment:
in case of 200 VPCNetworkConfig CR, each time need to loop to find the default one, is it possible we use variable for the default one?

Reply:
@zhengxiexie  the default nc name is not always default, the purpose of this patch is to use annotations on nc cr to get the default nc name.
@lxiaopei , agree, that is my first thought. i will change it according to your suggestion.